ddl/ddl.go

```diff
@@ -466,13 +472,51 @@ func (d *ddl) newDeleteRangeManager(mock bool) delRangeManager {
 	return delRangeMgr
 }
 
+func (d *ddl) readyForConcurrencyDDL() {
+	workerFactor := func(tp workerType) func() (pools.Resource, error) {
+		return func() (pools.Resource, error) {
+			wk := newWorker(d.ctx, tp, d.sessPool, d.delRangeMgr, d.ddlCtx, true)
+			sessForJob, err := d.sessPool.get()
+			if err != nil {
+				return nil, err
+			}
+			if tp == addIdxWorker {
+				sessForJob.SetDiskFullOpt(kvrpcpb.DiskFullOpt_AllowedOnAlmostFull)
+			} else {
+				sessForJob.SetDiskFullOpt(kvrpcpb.DiskFullOpt_NotAllowedOnFull)
```

Comment:
Why?

Reply:
This is the wrong code. I think it should be set as ```DiskFullOpt_AllowedOnAlmostFull```. If the ```AddIndex``` is running, it will be changed to ```DiskFullOpt_NotAllowedOnFull```.